zeek/compiler/glue-compiler.cc

```diff
@@ -760,6 +760,8 @@ bool GlueCompiler::compile() {
     auto import_ = hilti::declaration::ImportedModule(ID("zeek_rt"), std::string(".hlt"));
     init_module.add(std::move(import_));
 
+    auto preinit_body = hilti::builder::Builder(_driver->context());
```

Comment:
We could move this down before the loop using it.

Reply:
prefer to leave where it is as other part could later be adding to it to.